admin/broadleaf-open-admin-platform/src/main/resources/open_admin_style/js/admin/vendor/jquery.fluidbox.js

```diff
@@ -144,7 +139,12 @@
 						} else {
 							data.imgScale = $w.width()*settings.viewportFill/$img.width();
 						}						
-					}
+					};
+
+					var $img	= $fbItem.find('img'),
```

Comment:
Same as the other comment. Just add some block comments indicating that this was changed and why it was changed